### PR TITLE
EY-4270 Logge js feil i frontender

### PR DESCRIPTION
**Pull request by sebassonav**

### CODE REVIEW

File: apps/etterlatte-node-server/src/routers/loggerRouter.ts [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+    columnno: number
+    message: any
+    error: any
+}
+
+function isStackInfoValid(numbers: IStackLineNoColumnNo): boolean {
+    return numbers.lineno > 0 && numbers.columnno >= 0
+}
+
+const sourcemapLocation = '/app/client/assets'
```
[the remark this answers is not in the document]
Comment (sebassonav) [reply apps/etterlatte-node-server/src/routers/loggerRouter.ts]: https://github.com/navikt/pensjon-etterlatte/pull/1372/commits/e0995c1867a6513bb58ec75f006a0757e7d4e909

File: apps/barnepensjon-ui/src/ErrorBoundary.tsx [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+import styled from 'styled-components'
+import { Alert } from '@navikt/ds-react'
+import { Link } from 'react-router-dom'
+
+type Props = {
+    children: React.JSX.Element
+}
+
+class ErrorBoundary extends React.Component<Props, { hasError: boolean }> {
+    constructor(props: Props) {
```
Comment (emilps) [inline apps/barnepensjon-ui/src/ErrorBoundary.tsx:13]: Bevisst bruk av gammel react syntax?

File: apps/etterlatte-node-server/src/auth/session.ts [hunk 3]
```diff
@@ -1,7 +1,7 @@
 import { Request, RequestHandler, Response } from "express";
 import jwt from "jsonwebtoken";
 import { getHeaderTokenReq } from '../proxy'
-import logger from "../monitoring/logger";
+import {logger} from "../monitoring/logger";
```
Comment (emilps) [inline apps/etterlatte-node-server/src/auth/session.ts:4]:
```suggestion
import { logger } from "../monitoring/logger";
```

File: apps/barnepensjon-ui/src/ErrorBoundary.tsx (hunk 2 above)
Comment (sebassonav) [reply apps/barnepensjon-ui/src/ErrorBoundary.tsx:13]: Ja

**Comment by sebassonav** [comment]
Se eks her https://logs.adeo.no/app/r?l=DISCOVER_SINGLE_DOC_LOCATOR&v=8.13.4&lz=N4IglgdgJgpgHiAXCAnANhmgLADgMYAMAtCjgQIZECMVMKROAzBUTFQOwBGnU2nWUTiAA0IAE4B7AO4BJaPCQgAdFADORCOQBuAGwkBzdQCYCRrEoI4LOIgTtGzI8dJlRFAcXcFVAOQAOAKIAQgAaWKpSQVoQANZS7gD6Ek5iMABmMGKpYooA9OR%2BfrlQYKp4ElqZAMS5APwJ%2BgC8ABQALmAAtjCIzWmSHYgQ0tQArB3CrRKD0gCUMwBkCeQt5ToArh0QqogAhM06MJU6wl2qquT6MMIwEFp4OuRnwgV%2BOmB45O0SEMJ%2BElAzYSQWBwRAAcnQmFwhBIZEoNDoDGYlDYXB4fAEnDBwgAjmtMgBPHoPCD6NYXbrrPA3K54wngzSnPzkamIGCtVqZB4cmAAUhMAEEfAARfkEG53B5nRA4sVC0UmA5HRABLISMRguYgAC%2BQA

File: apps/omstillingsstoenad-ui/src/utils/logger.tsx [hunk 4]
```diff
@@ -1,10 +1,94 @@
...
+        const data = { type: 'info', data: info, jsonContent: { ...defaultContext } }
+        loggFunc(data)
+                .catch((err) => {
+                    console.error('Unable to log info message: ', data, ' err: ', err)
+                })
+    },
+    generalError: (info: object) => {
+        const data = { type: 'error', data: info, jsonContent: { ...defaultContext } }
+        loggFunc(data)
+                //.then(() => store.dispatch(loggError(data)))
```
Comment (emilps) [inline apps/omstillingsstoenad-ui/src/utils/logger.tsx:61]: Trengs denne delen av koden?

**Review by emilps — APPROVED** [review]

File: apps/omstillingsstoenad-ui/src/utils/logger.tsx (hunk 4 above)
Comment (sebassonav) [reply apps/omstillingsstoenad-ui/src/utils/logger.tsx]: Fjernet

**Pull request merged**